docs/v8-migration-api.md

```diff
@@ -77,6 +77,37 @@ Check out the [v8-migration guide](v8-migration.md) for other user-level changes
     - For `node.parameters` in Babel 7, use `node.params` in Babel 8
     - For `node.typeAnnotation` in Babel 7, use `node.returnType` in Babel 8
 
+![medium](https://img.shields.io/badge/risk%20of%20breakage%3F-medium-yellow.svg)
+
+- Split `typeParameter` of `TSMappedType` ([#16733](https://github.com/babel/babel/pull/16733)).
+
+  For a `TSMappedType` node, the `typeParameter` attribute is split into `key` and `constraint` attributes.
```

Comment:
Maybe lets mention that it is to align with the `@typescript-eslint` AST for TS nodes?